integration-tests/it-common-service/pom.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We need to add the Apache License header here.

Reply:
done